util/cinc-install-1.1.0.sh

```diff
@@ -259,6 +259,26 @@ do_download() {
   unable_to_retrieve_package
 }
 
+# get_region INSTANCE_METADATA_FILE
+# get region from metadata
+get_region() {
```

Comment:
We don't need to rename this file, we can just add a comment near to top and keep track of the version. See internal CR for how we will upload the file